kokoro failed on missing nullability annotations.

```
In file included from components/BottomSheet/src/MaterialBottomSheet.h:17:
components/BottomSheet/src/MDCBottomSheetController.h:66:37: error: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Werror,-Wnullability-completeness]
@property(nonatomic, copy) NSString *scrimAccessibilityLabel;
                                    ^
```

--- a/components/BottomSheet/src/MDCBottomSheetController.h
+++ b/components/BottomSheet/src/MDCBottomSheetController.h
@@ -63,14 +63,14 @@
 
  Defaults to @c nil.
  */
-@property(nonatomic, copy) NSString *scrimAccessibilityLabel;
+@property(nullable, nonatomic, copy) NSString *scrimAccessibilityLabel;
 
 /**
  The @c accessibilityHint value of the dimmed scrim view.
 
  Defaults to @c nil.
  */
-@property(nonatomic, copy) NSString *scrimAccessibilityHint;
+@property(nullable, nonatomic, copy) NSString *scrimAccessibilityHint;
 
 /**
  The @c accessibilityTraits of the dimmed scrim view.

--- a/components/BottomSheet/src/MDCBottomSheetPresentationController.h
+++ b/components/BottomSheet/src/MDCBottomSheetPresentationController.h
@@ -76,14 +76,14 @@
 
  Defaults to @c nil.
  */
-@property(nonatomic, copy) NSString *scrimAccessibilityLabel;
+@property(nullable, nonatomic, copy) NSString *scrimAccessibilityLabel;
 
 /**
  The @c accessibilityHint value of the dimmed scrim view.
 
  Defaults to @c nil.
  */
-@property(nonatomic, copy) NSString *scrimAccessibilityHint;
+@property(nullable, nonatomic, copy) NSString *scrimAccessibilityHint;
 
 /**
  The @c accessibilityTraits of the dimmed scrim view.

--- a/components/BottomSheet/src/MDCBottomSheetTransitionController.h
+++ b/components/BottomSheet/src/MDCBottomSheetTransitionController.h
@@ -64,14 +64,14 @@
 
  Defaults to @c nil.
  */
-@property(nonatomic, copy) NSString *scrimAccessibilityLabel;
+@property(nullable, nonatomic, copy) NSString *scrimAccessibilityLabel;
 
 /**
  The @c accessibilityHint value of the dimmed scrim view.
 
  Defaults to @c nil.
  */
-@property(nonatomic, copy) NSString *scrimAccessibilityHint;
+@property(nullable, nonatomic, copy) NSString *scrimAccessibilityHint;
 
 /**
  The @c accessibilityTraits of the dimmed scrim view.

